packages/components/organisms/f-footer/src/components/LinkList.vue

```diff
@@ -70,7 +70,7 @@ export default {
 
     data () {
         return {
-            panelCollapsed: false,
+            panelMobileCollapsed: true,
```

Comment:
Can we leave the name `panelCollapsed` as it is used not only on mobiles but on tablets too, and basically can be used on any width if needed

Reply:
ok